src/licensedcode/data/licenses/saxix-mit.LICENSE

```diff
@@ -2,12 +2,12 @@
 key: saxix-mit
 short_name: Stefano Apostolico MIT License
 name: Stefano Apostolico MIT License
-category: Permissive
+category: Proprietary Free
 owner: Stefano Apostolico
 homepage_url: https://github.com/saxix/django-adminactions/blob/develop/LICENSE
+notes: this is the MIT license text with an extra clause wrt. commercial usage notification.
 spdx_license_key: LicenseRef-scancode-saxix-mit
 minimum_coverage: 95
```

Comment:
I would move this to 100 and create a rule with these lines https://github.com/nexB/scancode-toolkit/pull/3778/files#diff-fba6a45f6fe77d22e39582632b7061815e79911610fe43ec50028bb8a64a8428R15-R33 (minus the MIT License line) and make the extra clause a key phrase.